backend/tests/test_participants_list.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+
+from backend.models import Participant
+
+
+def test_get_participants_when_logged_in(
+        client, access_token, add_participants
+     ):
+    """Test get participants list for logged in user."""
+    rv = client.get(
+        '/participants',
+        headers={'Authorization': 'Bearer {}'.format(access_token)}
+    )
+    response = rv.get_json()
+    assert rv.status == "200 OK"
+    assert len(response["participants"]) == 10
+
+
+def test_get_participants_with_empty_db(client, access_token):
+    """Test get participants list when no participant in db."""
+    rv = client.get(
+        '/participants',
+        headers={'Authorization': 'Bearer {}'.format(access_token)}
+    )
+    response = rv.get_json()
+    assert rv.status == "404 NOT FOUND"
```

Comment:
Shouldn't that be 200 with 0 participants?

Reply:
In my opinion there should not occur a situation where there are 0 participants.
At the very beginning all Code of Poznań users should be added as participant and participants_list should return all of them.

Reply:
If you can do something with the application, someone will do that. ;-)

Reply:
It should be `200 OK` here. That's a bug in the application and I created a separate issue to solve that: #130

Reply:
As it was error in my endpoint (get participants list), I corrected ParticipantsList endpoint:
pull request #139: changed response and http code (fixes #130)